server/lorax_server/models/flash_causal_lm.py

```diff
@@ -956,6 +956,13 @@ def __init__(
         config = config_cls.from_pretrained(model_id, revision=revision, trust_remote_code=trust_remote_code)
         config.quantize = quantize
 
+        from lorax_server.utils import paged_attention
+        if paged_attention.is_fp8_supported() and config.quantize and config.quantize.endswith('_kv'):
```

Comment:
Should be a utility function, since same code is used elsewhere in the mistral modeling.

Reply:
Yes, moved the import to the top. Server tests were breaking with the error `could not import vllm` when at top so had it here, but now that `is_fp8_supported` is in `torch_utils` even this shouldn't happen.